matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/data/PasswordLoginParams.kt

```diff
@@ -30,6 +30,7 @@ internal data class PasswordLoginParams(
         @Json(name = "identifier") val identifier: Map<String, String>,
         @Json(name = "password") val password: String,
         @Json(name = "type") override val type: String,
+        @Json(name = "refresh_token") override val refreshToken: Boolean,
```

Comment:
I would rename the `val` `withRefreshTokenSupport` for clarity. We may find a better name.

Reply:
`enableRefreshTokenAuth`?